### ISSUE TITLE
{ controls: false } options does not hide controls in fullscreen on iOS

### ISSUE THREAD

**Issue by gonzaloserrano** [nchutchind/cordova-plugin-streaming-media#174]
**What version of Streaming-Meda-Cordova-Plugin are you using?**

2.2.0

**What version of Cordova are you using?**

8.0.0

**What devices are affected?**

iPhone Xs

**Please describe the issue in detail, with relevant code samples**

Executed this code:
```js
const videoUrl = 'https://www.radiantmediaplayer.com/media/bbb-360p.mp4';
window.plugins.streamingMedia.playVideo(videoUrl, { controls: false });
```
to play the video.

**What did you expect to happen?**

Video is played in fullscreen without controls.

**What actually happened?**

Video is played in fullscreen with controls.

**Comment by JoseGeorges8** [comment]
Just had that happening to me as well. They do hide in the Android build but not on the iOS build

**Comment by shamilovtim** [comment]
I'll investigate as soon as I have moment. Probably next week

**Comment by webteambel** [comment]
hello,
is there any solution for this problem ?

**Comment by shamilovtim** [comment]
I'll look at this as soon as possible, sorry for the delay.

**Comment by shamilovtim** [comment]
@webteambel do you see this problem on an iphone Xs as well, or is it an emulator or different device?

**Comment by webteambel** [comment]
@shamilovtim I use an Iphone 6s , Os 12.1.4

**Comment by JoseGeorges8** [comment]
Is this still an issue?

**Comment by goedzo** [comment]
Yes it's still an issue, just test on iphone X. Still shows controls.

My options parameters are:
`        var options = {
        successCallback: function () {
            app_playingVideo=false;
            console.log("Video was closed without error.");
        },
        errorCallback: function (errMsg) {
            app_playingVideo=false;
            console.log("Video was with an error.");
            console.log(errMsg);
        },
        orientation: 'portrait',
        controls:false
        };
`

**Comment by Hadrhune** [comment]
Yep, still an issue on Iphone 6, iOS 12.  Shows video controls even with <video controls="false" ... >
Used as a video background it's quite tedious to have controls popping up when the video starts. Does anybody have a workaround to solve this?
